src/content/docs/reference/tables/requests.mdx

```diff
@@ -3,8 +3,158 @@ title: Requests table
 description: Reference docs for the httparchive.all.requests table
 ---
 
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
 [`httparchive.all.requests`](https://console.cloud.google.com/bigquery?ws=!1m5!1m4!4m3!1shttparchive!2sall!3srequests) is a partitioned and clustered table containing one row per request per page tested in the HTTP Archive. Pages are tested on a monthly basis and as of April 2022, both the root page and one secondary page are tested.
 
+## Example queries
+
+Here are some common operations you can perform with the `requests` table.
+
+### Count the pages crawled
+
+<Tabs>
+  <TabItem label="Query">
+
+  ```sql
+  /* This query will process 85 GB when run. */
+  SELECT
+    client,
+    is_root_page,
+    count(0) AS requests_total
+  FROM `httparchive.all.requests`
+  WHERE date = "2024-05-01"
+  group by client, is_root_page
+  ```
+
+  </TabItem>
+  <TabItem label="Results">
+
+  client | is_root_page | requests_total
+  -- | -- | --
+  mobile | true | 1517364094
+  desktop | true | 1299394354
+  mobile | false | 1216156430
+  desktop | false | 1093804725
+
+  </TabItem>
+</Tabs>
+
+### Size of requests served
+
+<Tabs>
+  <TabItem label="Query">
+
+  Let's check the size of individual requests served from websites across the entire dataset. To do this, we'll be using the respBodySize summary metric. This metric represents the size of the response payload in bytes. Since 1 byte is very granular, we'll divide by 1024 to get to 1 KB and then by 100 so that we are looking at this data with bin sizes of 100KB. We'll also wrap this in a ROUND() function to remove the decimal points and then multiply the result by 100. Using this technique, 1234567 bytes would be rounded to a bin of 1200 KB.
+
+  ```sql
+  /* This query will process 26 GB when run. */
+  WITH requests AS (
+    SELECT
+      ROUND(CAST(JSON_VALUE(summary, '$.respBodySize') AS INT64)/1024/100)*100 AS responseSize100KB,
+      COUNT(0) OVER () AS total_requests
+    FROM
+      `httparchive.all.requests` TABLESAMPLE SYSTEM (1 PERCENT)
+    WHERE
+      date = '2024-06-01'
+      AND client = 'desktop'
+      AND is_root_page
+      AND ROUND(CAST(JSON_VALUE(summary, '$.respBodySize') AS INT64)/1024/100)*100 > 0
```

Comment:
This drops all resources under 50 KB that get rounded down to 0. Is that intentional? Or if you're trying to drop 0 byte responses, we could do something like:

```sql
AND CAST(JSON_VALUE(summary, '$.respBodySize') AS INT64) > 0
```

Reply:
Thanks. It needs to filter out empty responses.

Reply:
0kb group also doesn't make sense, so I'll CEILING them up.